packages/message-encryption/src/symmetric.spec.ts

```diff
@@ -1,13 +1,17 @@
 import { IProtoMessage } from "@waku/interfaces";
-import { contentTopicToPubsubTopic } from "@waku/utils";
+import { contentTopicToPubsubTopic, RoutingInfo } from "@waku/utils";
 import { expect } from "chai";
 import fc from "fast-check";
 
 import { getPublicKey } from "./crypto/index.js";
 import { createDecoder, createEncoder } from "./symmetric.js";
 
-const contentTopic = "/js-waku/1/tests/bytes";
-const pubsubTopic = contentTopicToPubsubTopic(contentTopic);
+const testContentTopic = "/js-waku/1/tests/bytes";
+const testPubsubTopic = contentTopicToPubsubTopic(testContentTopic);
```

Comment:
same nit here, we don't need `testPubsubTopic` as it is calculated in `RoutingInfo`
also, that means we don't need `contentTopicToPubsubTopic` as a inutility, I'll remove it eventually